addons/web/static/tests/core/utils/sortable_list_tests.js

```diff
@@ -0,0 +1,965 @@
+                    elements: ".item",
+                    nest: true,
+                    onDragStart({ element }) {
+                        assert.step("start");
+                        assert.strictEqual(element.id, "dragged");
+                        this.firstMove = true;
+                    },
+                    onMove({ element, previous, next, parent, prevPos }) {
+                        if (this.firstMove) {
+                            assert.step("move 1");
+                            assert.strictEqual(element.id, "dragged");
+                            assert.strictEqual(previous.id, "noChild");
+                            assert.strictEqual(next.id, "parent");
+                            assert.notOk(parent);
+                            assert.strictEqual(prevPos.previous.id, "dragged");
+                            assert.strictEqual(prevPos.next.id, "noChild");
+                            assert.notOk(prevPos.parent);
+                            this.firstMove = false;
+                        } else {
+                            assert.step("move 2");
+                            assert.strictEqual(element.id, "dragged");
+                            assert.notOk(previous);
+                            assert.strictEqual(next.id, "child");
+                            assert.strictEqual(parent.id, "parent");
+                            assert.strictEqual(prevPos.previous.id, "noChild");
+                            assert.strictEqual(prevPos.next.id, "parent");
+                            assert.notOk(prevPos.parent);
+                        }
+                    },
+                    onDrop({ element, previous, next, parent }) {
+                        assert.step("drop");
+                        assert.strictEqual(element.id, "dragged");
+                        assert.notOk(previous);
+                        assert.strictEqual(next.id, "child");
+                        assert.strictEqual(parent.id, "parent");
+                    },
+                    onDragEnd({ element }) {
+                        assert.step("end");
+                        assert.strictEqual(element.id, "dragged");
+                    },
+                });
+            }
+        }
+
+        SortableList.template = xml`
+            <div t-ref="root">
+                <ul class="sortable_list">
+                    <li class="item" id="dragged">
+                        <span>dragged</span>
+                    </li>
+                    <li class="item" id="noChild">
+                        <span>noChild</span>
+                    </li>
+                    <li class="item" id="parent">
+                        <span>parent</span>
+                        <ul>
+                            <li class="item" id="child">
+                                <span>item</span>
+                            </li>
+                        </ul>
+                    </li>
+                </ul>
+            </div>
+        `;
```

Comment:
:socks: General Comment for the test: setting this as a static variable inside the component doesn't work in tests ?